Source/WebCore/html/ImageBitmap.h

```diff
@@ -66,6 +67,21 @@ struct ImageBitmapOptions;
 
 template<typename IDLType> class DOMPromiseDeferred;
 
+class DetachedImageBitmap {
+public:
+    DetachedImageBitmap(DetachedImageBitmap&&);
+    WEBCORE_EXPORT ~DetachedImageBitmap();
+    DetachedImageBitmap& operator=(DetachedImageBitmap&&);
+    size_t memoryCost() const { return m_bitmap->memoryCost(); }
```

Comment:
Maybe adding a function here to return an `OptionSet<ImageBitmapSerializationFlags>` can make the code in SerializedScriptValue.cpp a littler simpler.

Reply:
The detachedimagebitmap is not used to serialize a out-of-process transfer of imagebitmap. It's only used for in-memory transfer..